packages/kodeverk/src/aksjonspunktCodes.ts

```diff
@@ -94,7 +93,6 @@ enum AksjonspunktCode {
   OVERSTYRING_AV_OPPTJENINGSVILKARET = '6011',
   OVERSTYR_DEKNINGSGRAD = '6016',
   VURDER_OPPTJENINGSVILKARET = '5089',
-  OVERSTYR_LØPENDE_MEDLEMSKAPSVILKAR = '6012',
```

Comment:
Vil behandlingen som tidligere har løst dette aksjonspunktet feile nå?

Reply:
HMmm.. Det vil ikke fungere nei.

Reply:
Så det er bare valget for å overstyre som skal fjernes?

Reply:
Tidligere overstyrte verdier vil vel også vises da?

Reply:
Er det bare oppfylt/ikke oppfylt? Eller vises også en opphørsdato?
Overstyringen burde i hvertfall fjernes.